ui/tests/acceptance/secrets/backend/totp/key-test.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import authPage from 'vault/tests/pages/auth';
+import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
+import listPage from 'vault/tests/pages/secrets/backend/list';
+import { click, fillIn, currentURL, settled, waitUntil } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { v4 as uuidv4 } from 'uuid';
+import { spy } from 'sinon';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+module('Acceptance | totp key backend', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  const createVaultKey = async (keyName, issuer, accountName) => {
+    await fillIn(GENERAL.inputByAttr('name'), keyName);
+    await fillIn(GENERAL.inputByAttr('issuer'), issuer);
+    await fillIn(GENERAL.inputByAttr('accountName'), accountName);
+    await click('[data-test-totp-create]');
+  };
+
+  hooks.beforeEach(function () {
+    const flash = this.owner.lookup('service:flash-messages');
+    this.flashSuccessSpy = spy(flash, 'success');
+    this.flashDangerSpy = spy(flash, 'danger');
+
+    this.uid = uuidv4();
+    return authPage.login();
+  });
+
+  test('it views a key via menu option', async function (assert) {
```

Comment:
great job writing acceptance tests to test navigational items! this is exactly what we want to use these for